### PR TITLE
streamline working and warn about iseq caching

### PR DESCRIPTION
**Pull request by grosser**
@burke

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 **Beta-quality. See [the last section of this README](#trustworthiness).**
 
-Bootsnap is a library that plugs into a number of Ruby and (optionally) `ActiveSupport` and `YAML`
-methods to optimize and cache expensive computations. See [the How Does This Work section](#how-does-this-work) for more information.
+Bootsnap is a library that can plug into Ruby, `ActiveSupport` and `YAML`
```
Comment (jules2689) [inline README.md]: We want to emphasize that ActiveSupport is optional. Rails is not a dependency for Bootsnap

**Comment by grosser** [comment]
all of them are optional, so put them on equal footing ?

**Comment by jules2689** [comment]
> Bootsnap is a library that plugs into Ruby, with optional support for `ActiveSupport` and `YAML`

How about something like this?

**Comment by grosser** [comment]
sounds good, updated

**Pull request merged**